servers/physics_3d/godot_collision_solver_3d_sat.cpp

```diff
@@ -1632,14 +1637,16 @@ static void _collision_capsule_capsule(const GodotShape3D *p_a, const Transform3
 	const GodotCapsuleShape3D *capsule_A = static_cast<const GodotCapsuleShape3D *>(p_a);
 	const GodotCapsuleShape3D *capsule_B = static_cast<const GodotCapsuleShape3D *>(p_b);
 
-	real_t capsule_A_radius = capsule_A->get_radius();
-	real_t capsule_B_radius = capsule_B->get_radius();
+	real_t scale_A = p_transform_a.basis[0].length();
+	real_t scale_B = p_transform_b.basis[0].length();
+	real_t capsule_A_radius = capsule_A->get_radius() * scale_A;
+	real_t capsule_B_radius = capsule_B->get_radius() * scale_B;
 
 	// Get the closest points between the capsule segments
 	Vector3 capsule_A_closest;
 	Vector3 capsule_B_closest;
-	Vector3 capsule_A_axis = p_transform_a.basis.get_column(1) * (capsule_A->get_height() * 0.5 - capsule_A_radius);
-	Vector3 capsule_B_axis = p_transform_b.basis.get_column(1) * (capsule_B->get_height() * 0.5 - capsule_B_radius);
+	Vector3 capsule_A_axis = p_transform_a.basis.get_column(1) * (capsule_A->get_height() * scale_A * 0.5 - capsule_A_radius);
```

Comment:
This has the same problem as the sphere-capsule in that the transforms are being double-applied by both multiply by `scale_[A/B]`, and then multiplying by the scaled `p_transform_[a/b].basis`.

I got this to work by:
1. Removing the scaling of `capsule_[A/B]_radius` by the associated `scale_[A/B]`
2. Removing the scaling of `capsule_[A/B]->get_height()` by the associated `scale_[A/B]`
3. This causes the `capsule_[A/B]_axis` to be scaled only once so it's now in correct global coordinates
4. Adding the multiplies by `scale_[A/B]` into the `analytic_sphere_collision` call